mne/dipole.py

```diff
@@ -268,6 +268,28 @@ def plot_locations(self, trans, subject, subjects_dir=None,
             show_all, ax, block, show, scale=scale, color=color, fig=fig,
             title=title)
 
+    @verbose
+    def to_mni(self, subject, trans, subjects_dir=None,
+               verbose=None):
+        """Convert dipole location from head coordinate system to MNI coordinates.
+
+        Parameters
+        ----------
+        %(subject)s
+        %(trans_not_none)s
+        %(subjects_dir)s
+        %(verbose)s
+
+        Returns
+        -------
+        pos_mni : array, shape (n_pos, 3)
+            The MNI coordinates (in mm) of pos.
+        """
+        from .source_space import head_to_mni
```

Comment:
No need to nest this, it can go at the top. We only nest our own imports to avoid circular import problems (which shouldn't be the case here because `.source_space` is already imported from above)